src/main/java/net/pms/dlna/DLNAMediaDatabase.java

```diff
@@ -457,14 +536,31 @@ private Double toDouble(ResultSet rs, String column) throws SQLException {
 	public synchronized void insertData(String name, long modified, int type, DLNAMediaInfo media) {
 		Connection conn = null;
 		PreparedStatement ps = null;
+
+		boolean existingId = false;
+		if (media != null && media.getDatabaseFileId() > -1) {
+			existingId = true;
+		}
+
 		try {
 			conn = getConnection();
-			ps = conn.prepareStatement(
-				"INSERT INTO FILES(FILENAME, MODIFIED, TYPE, DURATION, BITRATE, WIDTH, HEIGHT, SIZE, CODECV, "+
-				"FRAMERATE, ASPECT, ASPECTRATIOCONTAINER, ASPECTRATIOVIDEOTRACK, REFRAMES, AVCLEVEL, BITSPERPIXEL, "+
-				"THUMB, CONTAINER, MODEL, EXPOSURE, ORIENTATION, ISO, MUXINGMODE, FRAMERATEMODE, STEREOSCOPY, "+
-				"MATRIXCOEFFICIENTS, TITLECONTAINER, TITLEVIDEOTRACK, VIDEOTRACKCOUNT, IMAGECOUNT, BITDEPTH) VALUES "+
-				"(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
+
+			if (existingId) {
```

Comment:
@SubJunk This looks like a bug. ID is the primary key, it a row with that ID already exist and you try to insert another one with the same ID, you will get a SQL error. What should be done in that case is update the existing row with information, something I've implemented in "my" changes (where the method is called ```insertOrUpdateData()```).

I made that because there are a need to update existing entries if the existing parsing is incomplete or something has changed. I don't understand how you've handled that here, do you have an update method as well?